### PR TITLE
App bar component

### PR DESCRIPTION
**Pull request by hokansom**
Built the AppBar view and added it to the home page. 

Also added ease to the login animation to improve the UX. 

Functionality will be added later.

### CODE REVIEW

File: client/src/components/AppBar/index.js [hunk 1]
```diff
@@ -0,0 +1,86 @@
...
+    cursor: pointer;
+  }
+`;
+
+const Username = styled.h3`
+  padding-top: ${spacing24};
+  margin: 0;
+`;
+
+const AppBar = ({ username = "Username", color = theme1 }) => {
```
Comment (josiahsrc) [inline client/src/components/AppBar/index.js]: Should this component be placed in the `components/common` folder instead?

File: client/src/resources/style-constants.js [hunk 2]
```diff
@@ -19,6 +19,10 @@ export const grey3 = "#e0e0e0"; // light gray
 export const link = "#2196f3"; // color for links
 export const highlightedLink = "#35baf6"; // color of links on hover
 export const errorRed = "#FF0002";
+export const eventOrange = "#FFA089";
+export const eventYellow = "#FFD958";
```
Comment (josiahsrc) [inline client/src/resources/style-constants.js:25]: Placing these colors here are fine for now, but I think eventually these colors should be stored on the backend and be pulled down as part of the "getEventInfo" response.

File: client/src/resources/style-constants.js [hunk 3]
```diff
@@ -32,6 +36,8 @@ export const fontSize64 = "3rem";
 /** Border Radiuses */
 export const borderRadius = "12px";
 
+/* Other */
+export const appBarHeight = "96px";
```
Comment (josiahsrc) [inline client/src/resources/style-constants.js:42]: I like this notion of adding the stylings for reusable-component to this file; good thinking.

**Review by josiahsrc — APPROVED** [review]
Very clean and simple. Left a few comments, but looks good overall 💯

File: client/src/resources/style-constants.js (hunk 2 above)
Comment (hokansom) [reply client/src/resources/style-constants.js:25]: Agreed, I just put it there for now so we have something to use.

**Review by Clotonervo — APPROVED** [review]

**Pull request merged**